audit/projects/kubernetes-public/service-accounts/kubernetes-public@appspot.gserviceaccount.com/description.json

```diff
@@ -0,0 +1,8 @@
+{
+  "displayName": "App Engine default service account",
```

Comment:
Maybe something to do with promoter?  Can't think of a reason

Reply:
I have been in app engine before (looking at gubernator), it's possible this is the same deal as everything else in https://github.com/kubernetes/k8s.io/issues/1675

I'm surprised clicking around on console auto-activates services.  Is it because our accounts are over-privileged? I'd rather see a "nope, you can't use this because the API isn't enabled for this project"